responsibleai/responsibleai/managers/causal_manager.py

```diff
@@ -494,5 +505,9 @@ def _load(path, rai_insights):
         inst.__dict__['_task_type'] = rai_insights.task_type
         inst.__dict__['_categorical_features'] = \
             rai_insights.categorical_features
+        inst.__dict__['_dropped_features'] = None
+        if rai_insights._feature_metadata:
+            inst.__dict__['_dropped_features'] = \
```

Comment:
I don't think you need to add dropped_features to the save/load path as it is just used for validation and not in actual computation of causal effects. Could you and possibly remove the above code.

Reply:
If we don't save/load dropped_features, there is a error where CausalManager has no attribute named dropped_features

Reply:
We should just change this to be feature_metadata instead of just passing the dropped_features inside feature_metadata.

Reply:
Good point, Updated, thanks!